test/test_device.cpp

```diff
@@ -27,9 +32,14 @@ std::string exec(const char *cmd) {
 
 TEST_CASE("Device can be constructed", "[yacx::device]") {
   try {
+    #if defined(_MSC_VER)
+    std::string name = exec("powershell -command \"wmic path win32_VideoController get name | "
+                            "Select-String -Pattern 'NVIDIA ([a-zA-Z0-9\-]+)' "
+                            "-AllMatches | % { $_.matches.groups[1].value }\"");
+    #else
```

Comment:
not exactly the same regex pattern, but I really don't remember why I used a lookahead stuff or w/e there. idk seems to work like this.

Reply:
don't think powershell regex supports lookahead/behind